renku/core/commands/dataset.py

```diff
@@ -670,17 +673,26 @@ def _update_datasets(client, names, creators, include, exclude, ref, delete, ext
         else:
             communication.echo("To update external files run update command with '--external' flag.")
 
-    if not possible_updates:
-        return
+    updated_files = []
+    deleted_files = []
 
-    updated_files, deleted_files = client.update_dataset_git_files(files=possible_updates, ref=ref, delete=delete)
+    if possible_updates:
+        updated_files, deleted_files = client.update_dataset_git_files(files=possible_updates, ref=ref, delete=delete)
+
+    if local_files:
+        updated, deleted = client.update_dataset_local_files(records=local_files, delete=delete)
+        updated_files.extend(updated)
+        deleted_files.extend(deleted)
 
     if deleted_files and not delete:
         communication.echo(
-            "Some files are deleted from remote. To also delete them locally "
-            "run update command with '--delete' flag."
+            "Some files are deleted in the repo. To also delete them locally run update command with '--delete' flag."
```

Comment:
```suggestion
            "Some files are deleted in the repo. To also delete them from dataset metadata run update command with '--delete' flag."
```

Might be more clear. otherwise users might be confused if they work with a local dataset.